composer.json

```diff
@@ -24,7 +24,11 @@
     "embed/embed": "^2.6"
   },
   "require-dev": {
-    "phpunit/PHPUnit": "~4.8"
+    "phpunit/PHPUnit": "~4.8",
+    "silverstripe/behat-extension": "dev-master",
+    "silverstripe/serve": "dev-improve-portability",
```

Comment:
I'm waiting for a PR to be merged, and the merge of that PR was contingent on this and the work in CMS being done. Perhaps we merge like this and I'll clean up once a new release is out?